app/models/assignment.rb

```diff
@@ -196,6 +191,10 @@ def main_submissions
     end
   end
 
+  def graders_ordered
+    graders.order("assignment_graders.order").to_a
```

Comment:
Should you drop the `.to_a` here, so that any future queries can still be queries instead of array operations?